snippets/algo_snippets.json

```diff
@@ -0,0 +1,26 @@
+{
+    "IsPrime": {
+        "prefix": [
+            "is_prime",
+            "prime"
+        ],
+        "body": [
+            "def is_prime(n: int) -> bool:",
```

Comment:
O(sqrt(n))

Reply:
> O(sqrt(n))

I just checked up until 104730, it works.
What a magic, this step by 6 thought?

Reply:
it should be applicable beyond 1,000,000

Reply:
> it should be applicable beyond 1,000,000

I read it from here https://primes.utm.edu/notes/faq/six.html

Reply:
check this out https://www.quora.com/Is-every-prime-number-other-than-2-and-3-of-the-form-6k%C2%B11-Is-this-a-proven-result-What-are-other-resources-about-it

Reply:
amazing, also update the vscode_ext_gen.py coz list is not needed actually. It will still function the same
do you want to add more to the algo snippet? if no then we can merge

Reply:
I want to add several algorithmic snippets like `DFS` `BFS` `Dijkstra's algorithm`, `Segment Tree`, `Genetic Algorithm` ... , lets start with a simple one and as time progresses based on the feedback lets add them

Reply:
> I want to add several algorithmic snippets like `DFS` `BFS` `Dijkstra's algorithm`, `Segment Tree`, `Genetic Algorithm` ... , lets start with a simple one and as time progresses based on the feedback lets add them

So should I push an update to the extension? Or you want to finish add them only update one time?

Reply:
Let's push it we can add them later add an label to show future snippets which will increase user experience

Reply:
> Let's push it we can add them later add an label to show future snippets which will increase user experience

Ok done